kitsune/search/v2/documents.py

```diff
@@ -15,33 +15,27 @@
 
 
 class WikiDocument(SumoDocument):
-    url = field.Keyword()
     updated = field.Date()
 
     product_ids = field.Keyword(multi=True)
     topic_ids = field.Keyword(multi=True)
-
-    # Document specific fields (locale aware)
-    title = SumoLocaleAwareTextField()
-    content = SumoLocaleAwareTextField(store=True, term_vector="with_positions_offsets")
-    summary = SumoLocaleAwareTextField(store=True, term_vector="with_positions_offsets")
-    keywords = SumoLocaleAwareTextField(multi=True)
-
     locale = field.Keyword()
-    current_id = field.Integer()
-    parent_id = field.Integer()
+    parent_id = field.Keyword()
     category = field.Keyword()
     slug = field.Keyword()
     is_archived = field.Boolean()
     recent_helpful_votes = field.Integer()
 
+    # Document specific fields (locale aware)
+    title = SumoLocaleAwareTextField()
```

Comment:
I think title should also be `store=True`d, so we don't have to parse the whole `_source` field when rending search results.

Reply:
I guess the same goes for `slug` and `locale`... maybe we can make this change later once we're actually rending search results